helm/private/registrar/registrar.go

```diff
@@ -145,6 +146,6 @@ func main() {
 	}
 
 	// Subprocess helm push
-	log.Println("Running helm push...")
-	runHelm(helmPath, []string{"push", chartPath, *registryURL}, helmPluginsPath, nil)
+	log.Printf("Running helm %s...", *pushCmd)
```

Comment:
Does this drop the trailing newline?

Reply:
Yes:
```
INFO: Running command line: bazel-bin/some/path/chart.push
2025/03/12 15:54:53 Running image pusher: /home/some/path/push_some_oci_image_publish.sh
2025/03/12 15:54:54 existing manifest: sha256:...
registry.gitlab.com/some/path@sha256:...
2025/03/12 15:54:55 existing manifest: 0.0.0-dev@sha256:...
2025/03/12 15:54:55 Running helm cm-push...
Pushing some-0.0.1.tgz to https://gitlab.com/api/v4/projects/42/packages/helm/stable...
Done.
```
I wondered, but then I noticed that the line a little further up didn't have any explicit newline either:
https://github.com/abrisco/rules_helm/blob/08e1704d5276631bbb571956f0947ffd54943935/helm/private/registrar/registrar.go#L141

Reply:
Fixed in the amended commit: f2e6a598f811b55ccfd0ec75377d3c6d73527925